packages/react/src/date/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { CdsDate as DateInput } from '@clr/core/date';
+
+import '@clr/core/forms/register';
```

Comment:
@coryrylan : Do I need to import the forms register here?

Reply:
Actually each of the form element's register calls forms register already. I'll update this!